cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1120,7 +1120,14 @@ function Restart-StackdriverLoggingAgent {
     Stop-Process -Force -PassThru -Id (Get-WmiObject win32_process |
       Where CommandLine -Like '*Stackdriver/logging*').ProcessId
   }
-  Start-Service StackdriverLogging
+
```

Comment:
Another thing to optimize is the TODO in line 1116. We can check periodically and return if the service was stopped successfully (so that we don't wait the full 10s)